types/react-native/index.d.ts

```diff
@@ -1,6 +1,9 @@
 // Type definitions for react-native 0.44
 // Project: https://github.com/facebook/react-native
-// Definitions by: Eloy Durán <https://github.com/alloy>, Fedor Nezhivoi <https://github.com/gyzerok>, HuHuanming <https://github.com/huhuanming>
+// Definitions by: Eloy Durán <https://github.com/alloy>
+//                 Fedor Nezhivoi <https://github.com/gyzerok>
+//                 HuHuanming <https://github.com/huhuanming>
+//                 Jeremi Stadler <https://github.com/jeremistadler>
```

Comment:
Please keep it on one line, otherwise the bot will not be able to parse this header.